website/layouts/partials/_default/vpick.html

```diff
@@ -0,0 +1,19 @@
+{{ $docs          := where site.Pages "Section" "versioned" }}
```

Comment:
Can we move this file to `layouts/partials/versioning` dir? (:

Reply:
Done